### DISCUSSION TITLE
Doing It Live Homework

### DISCUSSION THREAD

**Discussion by ZoeLeBlanc** [CultureAsData-UIUC/is310-fall-2024#5] [Assignments]
Please add a link to your `.github.io` website for the Doing It Live Homework, available here [https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/introducing-humanities-computing/08-intro-web#homework-doing-it-live](https://cultureasdata-uiuc.github.io/is310-fall-2024/materials/introducing-humanities-computing/08-intro-web#homework-doing-it-live). As a reminder, if you already have a `github.io` site, please share when you built it and what you updated 👍

**Comment by yosefstifanos** [comment]
https://yosefstifanos.github.io/

**Comment by ryanpearlman1** [comment]
https://ryanpearlman1.github.io

**Comment by jasonmorris8** [comment]
https://jasonmorris8.github.io/jasonmorris8.io/

**Comment by tinayang922** [comment]
https://tinayang922.github.io/

**Comment by OctaneCodes** [comment]
https://octanecodes.github.io/OctaneCodes/

**Comment by averyWebster** [comment]
https://averywebster.github.io/

**Comment by Cindy1226** [comment]
https://cindy1226.github.io/

**Comment by GloriaLiu930** [comment]
https://glorialiu930.github.io/

**Comment by haydnelm** [comment]
https://haydnelm.github.io/

**Comment by esr27** [comment]
https://esr27.github.io

**Comment by theajk2** [comment]
https://github.com/theajk2/theajk2.github.io/tree/main

**Comment by shoirathi** [comment]
https://shoirathi.github.io/

**Comment by AlanK3** [comment]
[alankang.info](https://alankang.info/)
I was already hosting a site thru Github, so I added the projects sections at the bottom of the page.

**Comment by ahmadsawwan** [comment]
https://ahmadsawwan.github.io/

**Comment by AG-Samaniego** [comment]
https://github.com/AG-Samaniego/AG-Samaniego.github.io

I actually made this a while ago using Jekyll-- It used to be extremely outdated with no pictures, but for now I've just updated it to have my current senior year information (instead of my info from sophomore yr when I was CS+Astro). I'm trying to figure out how to potentially put some blog cards or tables lol

**Comment by SamuelSokolovsky** [comment]
https://samuelsokolovsky.github.io

**Comment by yunchaecho2** [comment]
https://yunchaecho2.github.io

**Comment by elee33** [comment]
https://elee33.github.io

**Comment by lhayedimarucut** [comment]
https://lhayedimarucut.github.io

**Comment by frangara** [comment]
https://frangara.github.io/

**Comment by officalcharles** [comment]
https://officalcharles.github.io/

**Comment by Rebe-Rod** [comment]
https://rebe-rod.github.io/Rebe-Rod/

**Comment by chuzesta** [comment]
https://chuzesta.github.io

**Comment by JBrown-0** [comment]
https://JBrown-0.github.io/JBrown-0

**Comment by LucasWashor** [comment]
https://lucaswashor.github.io/

**Comment by kohtas2** [comment]
https://kshimbar.github.io/blog/
I made this website last year for another class, IS 445. I added a new blog post on that about this class!

**Comment by yukthas2** [comment]
https://yukthas2.github.io/

**Comment by henRotunno** [comment]
https://henrotunno.github.io

**Comment by adg77** [comment]
https://adg77.github.io

**Comment by divyaj5** [comment]
https://divyaj5.github.io/divyaj5/
